src/components/Login/Login.scss

```diff
@@ -9,9 +9,13 @@
     text-align: center;
     background-color: rgb(238, 248, 252);
   }
+  label{
```

Comment:
new line after each. class, add stylelint in the project, add husky and precommit hook not to allow push to repo withiut checking the code